### DISCUSSION TITLE
Request New Hardware Support: Rockchip | Add Rongpin King3399 support

### DISCUSSION THREAD

**Discussion by telanxu** [immortalwrt/immortalwrt#1246] [General]
### Describe the bug

Request New Hardware Support

Target: Rockchip
Sub-Target: RK3399

dts come from https://github.com/HIWLYF/build-linux/blob/master/boot/dts/rockchip/mainline/rk3399-rp-king3399.dts

Thanks a lot.

### ImmortalWrt version

master

### ImmortalWrt release

23.05.2

### ImmortalWrt target/subtarget

Rockchip/RK3399

### Device

Unsupported: Rongpin King 3399

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
PR is welcome.

**Comment by telanxu** [comment]
Sorry, I am not a coder, I can not create a workable source code for rongpin king3399, so, no PR for your reference... :-(
